src/modules/app-configuration/app-configuration.router.ts

```diff
@@ -0,0 +1,38 @@
+import { PrivateMetadataAppConfigurator } from "./app-configurator";
+import { createSettingsManager } from "./metadata-manager";
+import { logger as pinoLogger } from "../../lib/logger";
+import { appConfigInputSchema } from "./app-config-input-schema";
+import { GetAppConfigurationService } from "./get-app-configuration.service";
+import { router } from "../trpc/trpc-server";
+import { protectedClientProcedure } from "../trpc/protected-client-procedure";
+
+export const appConfigurationRouter = router({
+  fetch: protectedClientProcedure.query(async ({ ctx, input }) => {
+    const logger = pinoLogger.child({ saleorApiUrl: ctx.saleorApiUrl });
+
+    logger.debug("appConfigurationRouter.fetch called");
+
+    return new GetAppConfigurationService({
+      apiClient: ctx.apiClient,
+      saleorApiUrl: ctx.saleorApiUrl,
+    }).getConfiguration();
+  }),
+  setAndReplace: protectedClientProcedure
+    // TODO: Update the permissions required to change the app settings
```

Comment:
is this comment on purpose, or left after copy-paste?

Reply:
Wasnt sure if additional permissions should be checked for the emails app. I'll leave just app management